careers/technical-engineer.md

```diff
@@ -60,14 +90,10 @@ A Principal Engineer is still acting in a very hands-on role, and as such, they
 
 **Responsibilities:**
 
-* Define strategic technical initiatives
-* Solve technical problems of the highest scope and complexity for the entire organization
-* Actively seek out and prioritize our toughest technical challenges with a goal of creating significant improvement
-* Define code quality standards and metrics
-* Provide mentorship to Senior and Staff Engineers to help them grow in their technical responsibilities
-* Ensure the safety and reliability of Packback's technology
-* Manage projects across multiple years
-
-## Credits
-
-Packback's engineering ladder was inspired by documentation published by [Rent the Runway](https://dresscode.renttherunway.com/blog/ladder) and [Kickstarter](https://kickstarter.engineering/the-kickstarter-engineering-and-data-team-ladder-96996c3b327) on their engineering ladders. Thanks to them for open-sourcing that information.
+- Define strategic technical initiatives
+- Solve technical problems of the highest scope and complexity for the entire organization
+- Actively seek out and prioritize our toughest technical challenges with a goal of creating significant improvement
+- Define code quality standards and metrics
+- Provide mentorship to Senior and Staff Engineers to help them grow in their technical responsibilities
+- Ensure the safety and reliability of Packback's technology
+- Manage projects across multiple years
```

Comment:
I'd personally prefer if these used active `-ing` verbs for consistency with other sections, but it's not a big deal either way.